portal-api/src/v1/facility-dates/timestamp.ts

```diff
@@ -0,0 +1,24 @@
+import { formatISO, isValid, set, startOfDay } from 'date-fns';
+
+// TODO: DTFS2-6994 remove this function. It seems to be unnecessary with date-fns
+export const formattedTimestamp = (timestamp: string) => {
+  const date = new Date(Number.parseInt(timestamp, 10));
```

Comment:
Note to make sure to `parseInt` before passing string into `new Date`. Wonder whether its worth adding a [linting rule](https://github.com/amzn/eslint-plugin-no-date-parsing) to check for this